lists2safebrowsing.py

```diff
@@ -109,7 +121,7 @@ def load_json_from_url(config, section, key):
 # https://web.archive.org/web/20160422212049/https://developers.google.com/safe-browsing/developers_guide_v2#Canonicalization
 def canonicalize(d):
   if (not d or d == ""):
-    return d;
```

Comment:
nit: It might be good to do this in a separate commit. It's a good cleanup, but it distracts from the real purpose of your pull request (i.e. to add the new lists).